Dockerfile

```
@@ -0,0 +1,61 @@
+# Run:
+#   docker run --rm -p 8080:8080 ghillie:local
+
+# =============================================================================
+# Build stage: Create wheel from source
+# =============================================================================
+FROM python:3.12-slim AS build
+
+WORKDIR /build
+
+# Install build dependencies
+RUN pip install --no-cache-dir --upgrade pip wheel setuptools
```

Hadolint (2.14.0)

[warning] Pin versions in pip. Instead of `pip install <package>` use `pip install <package>==<version>` or `pip install --requirement <requirements file>`

(DL3013)